packages/sdk-v2/package.json

```diff
@@ -26,7 +26,7 @@
     "@babel/preset-env": "^7.16.4",
     "@babel/preset-react": "^7.16.0",
     "@babel/preset-typescript": "^7.16.0",
-    "@react-native-async-storage/async-storage": "^1.17.10",
+    "@react-native-async-storage/async-storage": "1.17.10",
```

Comment:
17.11 has been released 3 days ago which compiles with >ES2020 and for us to work with it on UI we have to upgrade to react-scripts@5
using 17.10 for now is quicker I think